.clj-kondo/src/hooks/clojure/core.clj

```diff
@@ -140,24 +139,34 @@
   [s]
   (str/ends-with? s "!"))
 
+(defn- explicitly-safe? [qualified-symbol]
```

Comment:
Unrelated but I was debugging something here and I tried to make this code more explicit and easier to follow